src/Commands/AutoTimedCenterGear.cpp

```diff
@@ -0,0 +1,44 @@
+#include "AutoTimedCenterGear.h"
+#include "AutoLeftGear.h"
+#include "AutoDrive.h"
+#include "GearScore.h"
+#include "SetShifter.h"
+#include "SetAutoShifter.h"
+#include "Delay.h"
+#include "SetGearRoller.h"
+#include "../RobotMap.h"
+#include "SetSqueeze.h"
+#include "SetGear.h"
+#include "TimedDrive.h"
+#include "AutoTurn.h"
+
+AutoTimedCenterGear::AutoTimedCenterGear() {
+
+  // Add Commands here:
+	// e.g. AddSequential(new Command1());
+	//      AddSequential(new Command2());
+	// these will run in order.
+
+	// To run multiple commands at the same time,
+	// use AddParallel()
+	// e.g. AddParallel(new Command1());
+	//      AddSequential(new Command2());
+	// Command1 and Command2 will run in parallel.
+  AddSequential(new SetAutoShifter(Chassis::kAutoShiftOff));
+  AddSequential(new SetShifter(Chassis::kShifterLow));
+  AddSequential(new TimedDrive(1.5,-.8,-.8));
```

Comment:
This is more like AutoTimedDrive. Please add AutoTimedDrive and fix this to be AutoTimedCenterGear.